pyproject.toml

```diff
@@ -15,6 +15,7 @@ dependencies = [
   "numpy==1.*",
   "tqdm",
   "matplotlib",
+  "torch",
```

Comment:
We'll need to figure out the way to make `torch` dependency optional, right?

Reply:
Yes, I would deal with it along with the refactoring in upcoming PR.